appveyor.yml

```diff
@@ -1,5 +1,5 @@
 version: 1.0.{build}
-os: Visual Studio 2015
+os: Visual Studio 2017
```

Comment:
Is this really the only change you need to appveyor? Is building with MSBuild 14.0 still appropriate?